### PR TITLE
Document provenance checks

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,4 +1,122 @@
...
+Provenance Checks
+=================
+
+The provenance check is done against Pipenv and Pipenv.lock that are expected
+as an input. The output is a structured report (with metadata) that states
+issues found in the application stack. There are currently reported the
+following issues:
+
+1. ``ERROR``/``ARTIFACT-DIFFERENT-SOURCE`` - reported if a package/artifact **is** installed from a different package source index in comparision to the configured one
+2. ``INFO``/``ARTIFACT-DIFFERENT-SOURCE`` - reported if a package/artifact **can be** installed from a different package source index in comparision to the configured one
```
Comment (fridex) [inline README.rst]: I would probably rename this one not to create confusion with 1.

File: README.rst [hunk 2]
```diff
@@ -1,4 +1,122 @@
...
+
+Provenance Checks
+=================
+
+The provenance check is done against Pipenv and Pipenv.lock that are expected
+as an input. The output is a structured report (with metadata) that states
+issues found in the application stack. There are currently reported the
+following issues:
+
+1. ``ERROR``/``ARTIFACT-DIFFERENT-SOURCE`` - reported if a package/artifact **is** installed from a different package source index in comparision to the configured one
```
Comment (fridex) [inline README.rst]: TODO: document how to specify package source index

File: README.rst (hunk 1 above)
Comment (goern) [reply README.rst]: ja, this is confusing... isnt it `ARTIFACT-POSSIBLE-DIFFERENT-SOURCE` ?

**Comment by fridex** [comment]
@goern updated, added section on installation and links to Pipenv docs on how to specify indexes.

**Comment by fridex** [comment]
recheck

**Comment by fridex** [comment]
Codacy reports a new issue, but clicking on "New Issues" in the check dashboard shows "Woo! You don't have new issues.".

**Pull request merged**